pyproject.toml

```diff
@@ -20,7 +20,8 @@ packages = [
 [tool.poetry.dependencies]
 python = "^3.6 || ^3.7 || ^3.8"
 invoke = "^1.4.1"
```

Comment:
Mind moving `invoke` to `dev-dependencies`? AFAIK it's not a requirement for actually using this plugin.

Reply:
done